core/ops_builtin.rs

```diff
@@ -287,6 +287,10 @@ async fn op_read_all(
   Ok(vec)
 }
 
+//
+//
+//
+
```

Comment:
What is this for?

Reply:
I was debugging something else and it's a leftover.